pa11ycrawler/templates/index.html

```diff
@@ -18,8 +18,8 @@ <h2 class="hd-4">Pages Checked</h2>
     data-search="true">
     <thead>
         <tr>
-            <th data-field='Page Title' data-sortable="true">Page Title</th>
-            <th data-field='Results Link' data-sortable="true">Results Link</th>
+            <th data-field='Page' data-sortable="true">Page</th>
```

Comment:
While it is a page title, it links to the results for that page, which was my train of thought, but I guess you were thinking that its a "report" so that is implied.  I can get behind that.